orchestrator/go.mod

```diff
@@ -111,6 +111,7 @@ require (
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/power-devops/perfstat v0.0.0-20210106213030-5aafc221ea8c // indirect
 	github.com/russellhaering/goxmldsig v1.4.0
+	github.com/samply/golang-fhir-models/fhir-models v0.3.2
```

Comment:
we already have `github.com/zorgbijjou/golang-fhir-models/fhir-models` which we use everywhere, why need samply here?